### PR TITLE
Range filter

### PR DESCRIPTION
**Pull request by moonso**
Add range querys for gemini and vcf adapter

### CODE REVIEW

**Comment by moonso** [comment]
ping @dnil , @robinandeer , @guillermo-carrasco 
Ready for merge :grapes:

File: puzzle/plugins/vcf/mixins/variant_mixin.py [hunk 1]
```diff
@@ -74,8 +74,11 @@ def variants(self, case_id, skip=0, count=30, filters=None):
         """
         filters = filters or {}
         case_obj = self.case(case_id=case_id)
+        
         limit = count + skip
-
+        
+        print(filters)
```
Comment (robinandeer) [inline puzzle/plugins/vcf/mixins/variant_mixin.py]: debug?

**Comment by robinandeer** [comment]
Ser bra ut!

File: puzzle/plugins/vcf/mixins/variant_mixin.py (hunk 1 above)
Comment (moonso) [inline puzzle/plugins/vcf/mixins/variant_mixin.py]: Allways

Comment (moonso) [inline puzzle/plugins/vcf/mixins/variant_mixin.py]: Ska fixa

**Pull request merged**